src/Form/Form.js

```diff
@@ -587,11 +588,11 @@ class Form {
         const config = getInputConfig(input)
 
         const shouldDecorate = await config.shouldDecorate(input, this)
-        if (!shouldDecorate) return this
+        if (!shouldDecorate && !this.shouldShowCredentialsImportFlow) return this
 
         input.setAttribute(ATTR_AUTOFILL, 'true')
 
-        const hasIcon = !!config.getIconBase(input, this)
+        const hasIcon = !!config.getIconBase(input, this) && !this.shouldShowCredentialsImportFlow
```

Comment:
I feel like the `shouldShowCredentialsImportFlow()` method should be called from inside [`config.getIconBase`](https://github.com/duckduckgo/duckduckgo-autofill/blob/b629ed523510bd0e02611733f45a4e33f6ab4410/src/Form/inputTypeConfig.js#L100) and return an empty string.

Reply:
Yeah good point!

Reply:
Updated https://github.com/duckduckgo/duckduckgo-autofill/pull/631/commits/03366ad416641e15bf246cddafef8d9b5cbee473